### ISSUE TITLE
Don't chdir in libnethack

### ISSUE THREAD

**Issue by condnsdmatters** [facebookresearch/nle#132]
# TL;DR:

Currently the nethack object creates its own temporary directory and [libnethack will `cd` into it](https://github.com/facebookresearch/nle/blob/master/sys/unix/unixmain.c#L119). For NetHack this makes sense -- it just `cd`'s into `HACKDIR` and looks for all game files in `pwd` (either via its `dlb` method or directly).

For us, this is annoying: Whenever we create multiple NLEs it gets hairy (also need to delete our temporary directories, and need to cd out of them before!), cf #114.

We should instead change NetHack to not change `pwd` but take some global path instead which we would set when starting NLE.

**Comment by prematurelyoptimized** [comment]
Another fun side effect is that anything saved to the local directory (like trained models) is very difficult to find later.  Depending on the retention policy on /tmp, they might not even exist.
